test/lib.sh

```diff
@@ -35,7 +35,7 @@ add_cleanup() {
 }
 
 empty_repo() {
```

Comment:
Not sure if `bash` supports hyphens, but should we change these to kebab-case?

Reply:
As above: it does (but I had to try it to find out), but IMO no we shouldn't.